python/cudf/cudf/_lib/utils.pyx

```diff
@@ -221,6 +220,32 @@ def _index_level_name(index_name, level, column_names):
         return f"__index_level_{level}__"
 
 
+cdef columns_from_unique_ptr(
+    unique_ptr[table] c_tbl
+):
+    """Convert a libcudf table into list of columns.
+
+    Parameters
+    ----------
+    c_tbl : unique_ptr[cudf::table]
```

Comment:
c++ namespaces were included quite pervasively in docstrings. If we were to make that change, it should be a separate effort to this PR. When we start hitting that route, an obvious counter argument is that you start to run into many namings that has much similarity (`column` in c++ and `Column` in python). The scope resolution operator would serve as a clear visual hint that you are talking about c++, not python.

Reply:
Let's punt on this for this PR. I think the solution would be to define clearer namespaces so that we can clearly refer to things like
- Cython wrappers of libcudf C++ functions: libcudf.table.table
- Cython implementations (like column.pyx): cycudf.column.Column
- Python objects: ColumnBase
But that's for a future where we've decided on what those namespaces should be and have them cleanly implemented.